python/sidecar/chat.py

```diff
@@ -100,3 +101,4 @@ def ask_question(self, n_choices: int = 1, temperature: float = 0.7) -> dict:
         response = self.call_model(messages=messages, n_choices=n_choices, temperature=temperature)
         choices = self.prepare_choices_for_client(response=response)
         return choices
+        return choices
```

Comment:
Looks like this last line got doubled up?

Reply:
Yea, there are a couple more spots too -- something was up with my VS Code + vim plugin